engine-tests/src/test/java/org/terasology/math/Diamond3iIteratorTest.java

```diff
@@ -35,13 +35,13 @@
 
     @Test
     public void testZeroDistanceIteration() {
-        Iterator<Vector3ic> iter = Diamond3iIterable.region(new org.joml.Vector3i(),0).build().iterator();
+        Iterator<Vector3ic> iter = Diamond3iIterable.region(new org.joml.Vector3i(),1).build().iterator();
         assertEquals(Lists.newArrayList(new Vector3i()), Lists.newArrayList(iter));
     }
 
     @Test
     public void testOneDistanceIteration() {
-        Iterator<Vector3ic> iter = Diamond3iIterable.region(new Vector3i(),1).build().iterator();
+        Iterator<Vector3ic> iter = Diamond3iIterable.region(new Vector3i(),2).build().iterator();
```

Comment:
No! that's not a good idea.
The changes I suggested kept the unit tests unchanged and intact. You changed the API and now every usage of that method would need to be adjusted.
Besides, the API was fine - If I create a diamond with radius 1, I expect the coordinates to be (-1, 0, 0), (1, 0, 0), ...
This behavior needs to be kept, my changes only optimized the internals.